pkg/transport/transport.go

```diff
@@ -116,6 +124,32 @@ func (transport *Transport) DialAddress(ctx context.Context, address string, opt
 	return conn, Error.Wrap(err)
 }
 
+// DialNodeInsecure returns an insecure grpc connection without tls to a node.
+//
+// Use this method for communication with localhost. For example, with the inspector or debugging services.
+// Otherwise in most cases DialNode should be used for communicating with nodes since it is secure.
+func (transport *Transport) DialNodeInsecure(ctx context.Context, node *pb.Node, opts ...grpc.DialOption) (conn *grpc.ClientConn, err error) {
```

Comment:
since this doesn't even  node node id checking, you could change this to be DialAddressInsecure for the same unencrypted effect